### DISCUSSION TITLE
Q: import '@waline/client/dist/waline.css'不了 || Q: import '@waline/client/dist/waline.css' fails

### DISCUSSION THREAD

**Discussion by maifuyangluo** [walinejs/waline#2220] [Q&A] [answered]
### Describe the bug | 问题描述

您好，我想问下“node_modules/@waline/client/package.json” 文件中存在"exports": {
    "./dist/waline.css": "./dist/waline.css",
  },
我却导不出来。
爆这样的错：Module not found: Error: Package path ./dist/waline.css is not exported from package。

以下是我使用官方文档的代码
```
<template>
  <Waline :serverURL="serverURL" :path="path" />
</template>
<script setup>
import { Waline } from '@waline/client/component';
import '@waline/client/dist/waline.css'

import { computed } from 'vue';
import { useRoute } from 'vue-router';



const serverURL = 'https://maifuyangluo-blog.vercel.app/';
const path = computed(() => useRoute().path);
`</script>`
```

### Website URL | 问题网站

https://maifuyangluo-blog.vercel.app/

### Where your waline deploy? | 服务部署在哪里？

Vercel (Default)

### Where your comment data store? | 数据存储在哪里？

LeanCloud(https://leancloud.app)

---
### Describe the bug | Problem description

Hello, I would like to ask about the existence of "exports" in the "node_modules/@waline/client/package.json" file: {
    "./dist/waline.css": "./dist/waline.css",
  },
But I can't guide it.
This error occurred: Module not found: Error: Package path ./dist/waline.css is not exported from package.

Below is my code using the official documentation
```
<template>
  <Waline :serverURL="serverURL" :path="path" />
</template>
<script setup>
import { Waline } from '@waline/client/component';
import '@waline/client/dist/waline.css'

import { computed } from 'vue';
import { useRoute } from 'vue-router';



const serverURL = 'https://maifuyangluo-blog.vercel.app/';
const path = computed(() => useRoute().path);
`</script>`
```

### Website URL | Problem website

https://maifuyangluo-blog.vercel.app/

### Where your waline deploy? | Where is the service deployed?

Vercel (Default)

### Where your comment data store? | Where is the data stored?

LeanCloud(https://leancloud.app)

**Comment by Mister-Hope** [comment, marked as answer]
The docs are already updated
